apps/voucher/pages/api/lnurlw/callback/[id]/index.ts

```diff
@@ -0,0 +1,125 @@
+// pages/api/lnurlw/callback/[id].js
+import { decode } from "light-bolt11-decoder"
+
+import { getWithdrawLinkByK1Query, updateWithdrawLinkStatus } from "@/services/db"
+import { convertCentsToSats, getWalletDetails } from "@/utils/helpers"
+import { env } from "@/env"
+import { PaymentSendResult, Status, Wallet } from "@/lib/graphql/generated"
+import { escrowApolloClient } from "@/services/galoy/client/escrow"
+import { fetchUserData } from "@/services/galoy/query/me"
+import { getRealtimePriceQuery } from "@/services/galoy/query/get-real-time-price"
+import { lnInvoicePaymentSend } from "@/services/galoy/mutation/ln-invoice-payment-send"
+const { NEXT_PUBLIC_GALOY_URL } = env
+
+export default async function handler(req: any, res: any) {
+  if (req.method === "GET") {
```

Comment:
same here

Reply:
Shifted to app router, not needed anymore.